packages/the-org/src/index.ts

```diff
@@ -10,6 +10,9 @@ import liaison from './liaison';
 import projectManager from './projectManager';
 import socialMediaManager from './socialMediaManager';
 
+// Import the Polygon plugin to ensure it's registered in the system
+import '@elizaos/plugin-polygon';
+
 /**
```

Comment:
What's the reason for this we shouldn't need to touch other packages as this going to be its own stand alone plugin